README.md

```diff
@@ -43,7 +43,7 @@ pip install --src . -r requirements.txt
 
 After cloning this project and add project folder to PYTHONPATH. Then run following command to run SDK.
 ```shell
-$ python ./network_insight_sdk_generic_datasources/main.py -d <device> -m <model> -s <device_type> -i <ip-address> -u <username> -p <password> -o <output.zip>
+$ python ./network_insight_sdk_generic_datasources/main.py -d <device> -m <model> -s <device_type> -i <ip-address> -u <username> -p <password> -e <enablepwd> -o <output.zip>
```

Comment:
Can we use something which states that enable password is for enable mode password on switch? Just to make others understand purpose of this password.